  INFO root [2021-09-30 15:17:20,496] - 
  INFO root [2021-09-30 15:17:20,497] - -----------------------------------------------------------------------------------------


  INFO root [2021-09-30 15:17:20,498] - Resmgr::initialize: KBE_ROOT=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/
  INFO root [2021-09-30 15:17:20,498] - Resmgr::initialize: KBE_RES_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/res/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/scripts/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/res/
  INFO root [2021-09-30 15:17:20,498] - Resmgr::initialize: KBE_BIN_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/bin/server/
  INFO root [2021-09-30 15:17:20,507] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.31.145 ( 192.168.31.145:56080 )
  INFO root [2021-09-30 15:17:20,507] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.31.145:56080, SOMAXCONN=511.
  INFO root [2021-09-30 15:17:20,507] - server-configs:
  INFO root [2021-09-30 15:17:20,508] - 	internalTcpAddr : 192.168.31.145:56080
  INFO root [2021-09-30 15:17:20,508] - 	componentID : 4000
  WARN root [2021-09-30 15:17:20,508] - invalid UID(0) <= 0, please check UID for environment! automatically set to 12169.
  INFO root [2021-09-30 15:17:20,640] - Components::findLogger: found logger, addr:192.168.31.145:51387
  INFO root [2021-09-30 15:17:20,641] - Components::addComponent: reset orderLog, uid=12169!
  INFO root [2021-09-30 15:17:20,641] - Components::addComponent[logger], uid=12169, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2021-09-30 15:17:20,644] - ---- dbmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CEEA1830B25FC811C6B0172173A5FCD0. Config: Debug X64. Built: 14:53:18 Sep 29 2021. AppID: 4000. UID: 12169. PID: 17892 ----
  INFO root [2021-09-30 15:17:20,645] - Server Administrator: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz with 16710936kB RAM
  INFO root [2021-09-30 15:17:20,645] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-30 15:17:20,645] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2021-09-30 15:17:20,842] - Script::install(): is successfully, Python=(3.7.3 (default, Sep 29 2021, 14:52:32) [MSC v.1928 64 bit (AMD64)])!
  INFO root [2021-09-30 15:17:20,842] - Script::install(): KBExtra is successfully!
  INFO root [2021-09-30 15:17:21,142] - Dbmgr::initializeEnd: digest(3240ED17A2FC6BF486E613049A996360)
S_INFO root [2021-09-30 15:17:21,142] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2021-09-30 15:17:21,142] - TelnetServer server is running on port 32000
  INFO root [2021-09-30 15:17:21,142] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
 DEBUG root [2021-09-30 15:17:21,168] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:17:22,438] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
  INFO root [2021-09-30 15:17:22,438] - DBUtil::createInterface[0x9e75cfe5f8]: interface=default, dbtype=mysql, ip=23.105.212.231, port=3306, currdatabase=kbe, username=root, connected=yes.

  INFO root [2021-09-30 15:17:22,438] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-30 15:17:22,438] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2021-09-30 15:17:22,438] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:17:22,438] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:17:22,438] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:17:22,438] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:17:22,439] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
  INFO root [2021-09-30 15:17:22,733] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:22,733] - DBInterfaceMysql::checkErrors: not found Consumer table(dbInterface=default), reset kbe_* table...
  WARN root [2021-09-30 15:17:22,949] - DBInterfaceMysql::checkErrors: reset kbe_* table(dbInterface=default) end!
 DEBUG root [2021-09-30 15:17:23,482] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:17:23,482] - DBThread::onStart(): 0x292b73347d0!
 DEBUG root [2021-09-30 15:17:23,486] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:17:23,487] - DBThread::onStart(): 0x292b7334b10!
 DEBUG root [2021-09-30 15:17:23,487] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:17:23,487] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:17:23,487] - DBThread::onStart(): 0x292b7333ee0!
 DEBUG root [2021-09-30 15:17:23,487] - DBThread::onStart(): 0x292b7335670!
 DEBUG root [2021-09-30 15:17:23,487] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:17:23,487] - DBThread::onStart(): 0x292b7333fb0!
  INFO root [2021-09-30 15:17:23,785] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:17:23,871] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:17:23,877] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:23,877] - DBTask::process(): took 0.24 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255) not null,`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  INFO root [2021-09-30 15:17:24,031] - DBInterfaceMysql::query: successfully!
 DEBUG root [2021-09-30 15:17:24,032] - syncToDB(): Consumer->sm_autoLoad(tinyint not null DEFAULT 0).
  INFO root [2021-09-30 15:17:24,182] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:24,248] - DBTask::process(): took 0.61 seconds
sql:(insert into kbe_serverlog (heartbeatTime, isShareDB, serverGroupID) values(1632986244,0,12169) ON DUPLICATE KEY UPDATE heartbeatTime=1632986244,isShareDB=0)
  INFO root [2021-09-30 15:17:24,332] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:24,333] - DBTask::process(): took 0.70 seconds
sql:(delete from kbe_email_verification where logtime<1632982644 and type=3)
  INFO root [2021-09-30 15:17:24,341] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:17:24,437] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:17:24,644] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:24,744] - DBTask::process(): took 1.11 seconds
sql:(show index from tbl_Consumer)
  WARN root [2021-09-30 15:17:24,944] - DBTask::process(): took 1.31 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
 DEBUG root [2021-09-30 15:17:25,291] - DBInterfaceMysql::dropEntityTableFromDB: tbl_Account.
  INFO root [2021-09-30 15:17:25,477] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:17:25,478] - D:\UnrealEngineProject\kbengine-2.5.12\kbengine-2.5.12\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 4.34 seconds
  INFO root [2021-09-30 15:17:25,696] - ---- dbmgr is running ----
 DEBUG root [2021-09-30 15:17:25,697] - Components::process(): Request for the process of identity...
  WARN root [2021-09-30 15:17:31,908] - Forwarding logs to logger[192.168.31.145:51387/0/0/0]...
 ERROR root [2021-09-30 15:22:22,446] - DBInterfaceMysql::processException: Thread 0x292b93ff820 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:22:23,710] - DBInterfaceMysql::processException: Thread 0x292b9407910 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:22:38,644] - DBInterfaceMysql::processException: Thread 0x292b9410210 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:22:39,724] - DBInterfaceMysql::processException: Thread 0x292b93e4e90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:23:24,766] - DBInterfaceMysql::processException: Thread 0x292b93fcd90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:28:25,986] - DBInterfaceMysql::processException: Thread 0x292b93fcd90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:34:42,959] - DBInterfaceMysql::processException: Thread 0x292b9407910 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:34:52,537] - DBInterfaceMysql::processException: Thread 0x292b9410210 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:35:14,505] - DBInterfaceMysql::processException: Thread 0x292b93e4e90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:38:57,840] - DBInterfaceMysql::processException: Thread 0x292b93ff820 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:38:58,907] - DBInterfaceMysql::processException: Thread 0x292b93fcd90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:39:22,523] - DBInterfaceMysql::processException: Thread 0x292b9407910 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:47:15,762] - DBInterfaceMysql::processException: Thread 0x292b9410210 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:47:18,441] - DBInterfaceMysql::processException: Thread 0x292b93e4e90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:47:51,808] - DBInterfaceMysql::processException: Thread 0x292b93ff820 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:48:27,500] - DBInterfaceMysql::processException: Thread 0x292b93fcd90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:48:48,616] - DBInterfaceMysql::processException: Thread 0x292b9407910 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:54:24,618] - DBInterfaceMysql::processException: Thread 0x292b9410210 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:54:28,076] - DBInterfaceMysql::processException: Thread 0x292b93e4e90 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:54:49,195] - DBInterfaceMysql::processException: Thread 0x292b93ff820 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 15:55:36,979] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=628355403219, lastReceivedTime=628153123359, inactivityExceptionPeriod=599471224)
   OFF root [2021-09-30 15:55:36,984] - The following logs sent to logger failed:
  WARN root [2021-09-30 15:55:36,984] - ==> [2021-09-30 15:55:36 978] TCPPacketReceiver::processPendingEvents(192.168.31.145:51387): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2021-09-30 15:55:36,985] - ==> [2021-09-30 15:55:36 979] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=628355403219, lastReceivedTime=628153123359, inactivityExceptionPeriod=599471224)
  INFO root [2021-09-30 15:55:51,310] - 
  INFO root [2021-09-30 15:55:51,311] - -----------------------------------------------------------------------------------------


  INFO root [2021-09-30 15:55:51,311] - Resmgr::initialize: KBE_ROOT=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/
  INFO root [2021-09-30 15:55:51,312] - Resmgr::initialize: KBE_RES_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/res/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/scripts/;D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/server_assets/res/
  INFO root [2021-09-30 15:55:51,312] - Resmgr::initialize: KBE_BIN_PATH=D:/UnrealEngineProject/kbengine-2.5.12/kbengine-2.5.12/kbe/bin/server/
  INFO root [2021-09-30 15:55:51,321] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.31.145 ( 192.168.31.145:61140 )
  INFO root [2021-09-30 15:55:51,321] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.31.145:61140, SOMAXCONN=511.
  INFO root [2021-09-30 15:55:51,321] - server-configs:
  INFO root [2021-09-30 15:55:51,321] - 	internalTcpAddr : 192.168.31.145:61140
  INFO root [2021-09-30 15:55:51,321] - 	componentID : 4000
  WARN root [2021-09-30 15:55:51,321] - invalid UID(0) <= 0, please check UID for environment! automatically set to 12169.
  INFO root [2021-09-30 15:55:51,430] - Components::findLogger: found logger, addr:192.168.31.145:51763
  INFO root [2021-09-30 15:55:51,430] - Components::addComponent: reset orderLog, uid=12169!
  INFO root [2021-09-30 15:55:51,430] - Components::addComponent[logger], uid=12169, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2021-09-30 15:55:51,434] - ---- dbmgr Version: 2.5.10. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: CEEA1830B25FC811C6B0172173A5FCD0. Config: Debug X64. Built: 14:53:18 Sep 29 2021. AppID: 4000. UID: 12169. PID: 18220 ----
  INFO root [2021-09-30 15:55:51,434] - Server Administrator: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz with 16710936kB RAM
  INFO root [2021-09-30 15:55:51,434] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-30 15:55:51,435] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2021-09-30 15:55:51,619] - Script::install(): is successfully, Python=(3.7.3 (default, Sep 29 2021, 14:52:32) [MSC v.1928 64 bit (AMD64)])!
  INFO root [2021-09-30 15:55:51,619] - Script::install(): KBExtra is successfully!
  INFO root [2021-09-30 15:55:51,702] - Dbmgr::initializeEnd: digest(3240ED17A2FC6BF486E613049A996360)
S_INFO root [2021-09-30 15:55:51,702] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2021-09-30 15:55:51,702] - TelnetServer server is running on port 32000
  INFO root [2021-09-30 15:55:51,703] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
 DEBUG root [2021-09-30 15:55:52,234] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:55:53,486] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
  INFO root [2021-09-30 15:55:53,487] - DBUtil::createInterface[0xf75e55e558]: interface=default, dbtype=mysql, ip=23.105.212.231, port=3306, currdatabase=kbe, username=root, connected=yes.

  INFO root [2021-09-30 15:55:53,487] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2021-09-30 15:55:53,487] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2021-09-30 15:55:53,488] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:55:53,488] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:55:53,488] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:55:53,488] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
 DEBUG root [2021-09-30 15:55:53,488] - DBInterfaceMysql::attach: connect: 23.105.212.231:3306 starting...
  INFO root [2021-09-30 15:55:53,777] - DBInterfaceMysql::query: successfully!
 DEBUG root [2021-09-30 15:55:54,522] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:55:54,523] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:55:54,523] - DBThread::onStart(): 0x2abf162fc00!
 DEBUG root [2021-09-30 15:55:54,523] - DBThread::onStart(): 0x2abf1630830!
 DEBUG root [2021-09-30 15:55:54,524] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:55:54,525] - DBThread::onStart(): 0x2abf1630900!
 DEBUG root [2021-09-30 15:55:54,526] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:55:54,526] - DBThread::onStart(): 0x2abf1631870!
 DEBUG root [2021-09-30 15:55:54,728] - DBInterfaceMysql::attach: successfully! addr: 23.105.212.231:3306
 DEBUG root [2021-09-30 15:55:54,728] - DBThread::onStart(): 0x2abf162ff40!
  INFO root [2021-09-30 15:55:54,818] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:54,818] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:55,089] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:55:55,265] - DBTask::process(): took 0.59 seconds
sql:(insert into kbe_serverlog (heartbeatTime, isShareDB, serverGroupID) values(1632988555,0,12169) ON DUPLICATE KEY UPDATE heartbeatTime=1632988555,isShareDB=0)
  WARN root [2021-09-30 15:55:55,265] - DBTask::process(): took 0.59 seconds
sql:(show index from tbl_Consumer)
  INFO root [2021-09-30 15:55:55,266] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:55,301] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:55,446] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:55,599] - DBInterfaceMysql::query: successfully!
  INFO root [2021-09-30 15:55:55,624] - DBInterfaceMysql::query: successfully!
  WARN root [2021-09-30 15:55:55,624] - DBTask::process(): delay 1.13 seconds, try adjusting the kbengine[_defs].xml(numConnections) and MySQL(my.cnf->max_connections or innodb_flush_log_at_trx_commit)!
sql:(delete from kbe_email_verification where logtime<1632984955 and type=3)
  WARN root [2021-09-30 15:55:55,624] - DBTask::process(): took 0.71 seconds
sql:(delete from kbe_email_verification where logtime<1632984955 and type=3)
  WARN root [2021-09-30 15:55:55,896] - DBTask::process(): took 1.22 seconds
sql:(select distinct(serverGroupID) from kbe_entitylog)
  WARN root [2021-09-30 15:55:56,243] - D:\UnrealEngineProject\kbengine-2.5.12\kbengine-2.5.12\kbe\src\server\dbmgr\dbmgr.cpp:306: Profile scriptCall took 4.54 seconds
  INFO root [2021-09-30 15:55:56,522] - ---- dbmgr is running ----
 DEBUG root [2021-09-30 15:55:56,522] - Components::process(): Request for the process of identity...
  WARN root [2021-09-30 15:56:02,742] - Forwarding logs to logger[192.168.31.145:51763/0/0/0]...
 ERROR root [2021-09-30 16:06:26,813] - DBInterfaceMysql::processException: Thread 0x2abf3701b20 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 16:06:48,271] - DBInterfaceMysql::processException: Thread 0x2abf370cd30 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 16:10:01,457] - DBInterfaceMysql::processException: Thread 0x2abf36fe920 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 16:12:24,044] - DBInterfaceMysql::processException: Thread 0x2abf3703420 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 16:12:45,161] - DBInterfaceMysql::processException: Thread 0x2abf3700220 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 16:56:15,456] - DBInterfaceMysql::processException: Thread 0x2abf3701b20 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 17:56:15,395] - DBInterfaceMysql::processException: Thread 0x2abf370cd30 lost connection to database. Exception: Lost connection to MySQL server during query. Attempting to reconnect.
 ERROR root [2021-09-30 18:55:56,090] - DBInterfaceMysql::processException: Thread 0x2abf36fe920 lost connection to database. Exception: MySQL server has gone away. Attempting to reconnect.
